resources/js/Pages/Adminland/Employee/Archives/Show.vue

```diff
@@ -242,24 +257,55 @@ export default {
 
   data() {
     return {
-      loadingState: '',
+      dataReport: null,
+      form: useForm(),
+      refresh: _.debounce(() => this.doRefresh(), 1000),
     };
   },
 
+  computed: {
+    localReport() {
+      return this.dataReport ?? this.report;
+    }
+  },
+
+  mounted() {
+    this.dataReport = this.report;
+    this.refresh();
+  },
+
+  unmounted() {
+    this.refresh.cancel();
+  },
+
   methods: {
     submit() {
```

Comment:
Here you don't change the state of the loading button which triggers this method. You should change the state of  the loading state to `loading`, so the button starts to load.

Reply:
This was done by the `form.processing` property, but it changed, I have to fix it, thanks!